tests/ui/codegen/mismatched-data-layouts.stderr

```diff
@@ -0,0 +1,4 @@
+error: data-layout for target `mismatched-data-layout-7814813422914914169`, `e-m:e-i64:64-f80:128-n8:16:32:64-S128`, differs from LLVM target's `x86_64-unknown-none-gnu` default layout, `e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128`
```

Comment:
Is it possible to wildcard the expected DL here? Otherwise this test will fail on some LLVM versions.

Reply:
Fixed